### ISSUE TITLE
Links in Bootstrap projects aren't underlined

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1208]
Links in main content of Bootstrap projects aren't underlined (they should be for accessibility). This is done in the bootstrap css not our css but we should reverse this

**Comment by ronm123** [comment]
@FayCross ok with you to add text-underline-position: under; to this?

**Comment by FayCross** [comment]
@ronm123 sure

**Comment by ronm123** [comment]
done.
